current-sms-undeliverable-users-by-broadcast-id/lib/northstar.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+/**
+ * Imports.
+ */
+const Northstar = require('@dosomething/northstar-js');
+const logger = require('winston');
+const config = require('../config/lib/northstar');
+
+/**
+ * Setup.
+ */
+let client;
+
+/**
+ * @return {Object}
+ */
+module.exports.createNewClient = function createNewClient() {
+  const loggerMsg = 'northstar.createNewClient';
+
+  try {
+    client = new Northstar(config.clientOptions);
```

Comment:
Since this method is called `createNewClient`, should it return a new one each time (rather than a singleton)?